File: src/test/java/uk/co/eelpieconsulting/feedlistener/IdBuilderTest.java
package uk.co.eelpieconsulting.feedlistener;

import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertEquals;

public class IdBuilderTest {

    @Test
    public void canGenerateHumanReadableIdFromText() {
        assertEquals("the-quick-brown-fox", new IdBuilder().makeIdFor("The quick, brown fox."));
    }

}

File: src/test/kotlin/uk/co/eelpieconsulting/feedlistener/IdBuilderTest.kt
package uk.co.eelpieconsulting.feedlistener

import org.junit.jupiter.api.Test
import org.junit.jupiter.api.Assertions.assertEquals

class IdBuilderTest {
    @Test
    fun canGenerateHumanReadableIdFromText() {
        assertEquals("the-quick-brown-fox", IdBuilder().makeIdFor("The quick, brown fox."))
    }
}
